File: 프로그래머스/0/181861. 배열의 원소만큼 추가하기/배열의 원소만큼 추가하기.cs
using System;
using System.Collections.Generic;
public class Solution {
    public int[] solution(int[] arr) {
        List<int> result = new List<int>();
        int target =0;
        for(int i =0;i<arr.Length;i++)
        {
            target = arr[i];
            for(int j = 0 ; j < target; j++)
            {
                result.Add(target);
            }
        }
        return result.ToArray();
    }
}

File: 프로그래머스/0/181861. 배열의 원소만큼 추가하기/배열의 원소만큼 추가하기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr) {
    vector<int> answer;
    for(auto elem : arr)
    {
        for(int i = elem; i > 0; i--)
            answer.push_back(elem);
    }
    return answer;
}